cypher/cypher-docs/src/docs/dev/glossary.asciidoc

```diff
@@ -101,83 +101,83 @@ This section comprises a glossary of all the keywords -- grouped by category and
 |<<functions-pi, pi()>>                         | Trigonometric     | Returns the mathematical constant _pi_.
 |<<functions-point-cartesian,point() - cartesian 2D>>                    | Spatial           | Returns a point object, given two coordinate values in the cartesian coordinate system.
 |<<functions-point,point() - WGS 84>>                    | Spatial           | Returns a point object, given two coordinate values in the WGS 84 coordinate system.
-|<<functions-properties,properties()>>          | Scalar            | If the argument is a node or a relationship, the returned map is a map of its properties.
+|<<functions-properties,properties()>>          | Scalar            | Returns a map containing all the properties of a node or relationship.
 |<<functions-radians, radians()>>               | Trigonometric     | Converts degrees to radians.
-|<<functions-rand, rand()>>                     | Numeric           | Returns a random number in the range from 0 (inclusive) to 1 (exclusive), `[0, 1)`.
-|<<functions-range,range()>>                    | List              | Returns numerical values in a range.
-|<<functions-reduce,reduce()>>                  | List              | Runs an expression against individual elements of a list, storing the result of the expression in an accumulator.
-|<<functions-relationships,relationships()/rels()>>    | List              | Returns all relationships in a path.
-|<<functions-replace,replace()>>                | String            | Returns a string with the search string replaced by the replace string, replacing all occurrences.
-|<<functions-reverse,reverse()>>                | String            | Returns the original string reversed.
-|<<functions-right,right()>>                    | String            | Returns a string containing the right n characters of the original string.
-|<<functions-round, round()>>                   | Numeric           | Returns the numerical expression, rounded to the nearest integer.
-|<<functions-rtrim,rTrim()>>                    | String            | Returns the original string with whitespace removed from the right side.
-|<<functions-sign, sign()>>                     | Numeric           | Returns the signum of a number -- zero if the expression is zero, `-1` for any negative number, and `1` for any positive number.
-|<<functions-sin, sin()>>                       | Trigonometric     | Returns the sine of the expression.
-|<<functions-single,single()>>                  | Predicate         | Returns true if the predicate holds for exactly one of the elements in the list.
+|<<functions-rand, rand()>>                     | Numeric           | Returns a random floating point number in the range from 0 (inclusive) to 1 (exclusive); i.e. [0, 1).
+|<<functions-range,range()>>                    | List              | Returns a list comprising all integral values within a range bounded by a start and end value, where the difference between any two consecutive values is constant.
```

Comment:
The new version is obviously much more explicit and correct, but do we really need to be that explicit? It does make for a very long description. Are there any length policies for this table of descriptions? There are other places where I think detailed descriptions would bring more value (like the all(), none(), any() functions for example).

Reply:
Revert to orig in this instance